src/index.js

```diff
@@ -1,9 +1,26 @@
 'use strict';
 
 const NewsEngine = require('./news-engine');
+const config = require('../config');
+const fs =  require(`fs`);
+const uuid = require(`uuid`);
+const svg2png = require('svg2png');
 
 const numExamples = process.argv[2] || 30;
 const headlines = NewsEngine.generateHeadlines(numExamples);
 headlines.forEach(headline => {
 	console.log("\n * " + headline.text);
+
+	if(headline.tags && headline.tags.svg && headline.tags.svg.svgString) {
+		const outputfile = uuid() + '.png';
+		const outputPath = config.paths.tempDirectory + '/' + outputfile;
+
+		svg2png(new Buffer(headline.tags.svg.svgString))
```

Comment:
This is done async so all the images and the output for them pile up at the end of the run. Whether this is desirable or not is up for debate but it was the Easy(tm) way to do it so I did it for now.